docs/migrations.md

```diff
@@ -124,6 +124,10 @@ class Migration(migrations.Migration):
 The `forwards()` and `backwards()` functions are where any changes
 that need to happen to a model's data are made.
 
+!!! note
+    While backwards migrations are necessary in external libraries that we create, 
```

Comment:
Is "encouraged" a better word to use here than "necessary"?

Reply:
Perhaps? But I feel like any exceptions should be adequately explained (I would hate to encounter a library that I couldn't migrate backwards, personally). I like the stronger language, personally.